src/Administration/Resources/app/administration/src/module/sw-product/page/sw-product-detail/index.js

```diff
@@ -385,6 +385,36 @@ export default {
         currentPage() {
             return Shopware.Store.get('cmsPage').currentPage;
         },
+
+        languageRepository() {
+            return this.repositoryFactory.create('language');
+        },
+
+        language() {
+            return Shopware.Store.get('context').api.language;
+        },
+
+        translateFields() {
+            if (!this.product) {
+                return null;
+            }
+
+            return Object.keys(EntityDefinition.getTranslatedFields(this.product.getEntityName()));
+        },
+
+        ignoreFieldsValidation() {
```

Comment:
could be helpful to put a comment that translated fields of children language could inherited values from parent language